frontend/src/lib/components/PasswordStrengthMeter.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+  import t from '$lib/i18n';
+  import zxcvbn from 'zxcvbn';
+
+  // zxcvbn password strength is an int between 0 and 4 inclusive
+  // Feedback colors are red for bad passwords, yellow for poor, green for good
+  // Set threshholds for "bad" and "poor" by changing the values below
+  export let bad = 0;
+  export let poor = 2;
+
+  let score: 0|1|2|3|4;
+
+  function passwordStrengthColor(score: number): 'error' | 'warning' | 'success' {
+    if (score <= bad) return 'error';
```

Comment:
For tailwind and daisy classes we always have to include the whole class name in the code (i.e. you can't build it dynamically as you're doing in the `<progress>` element) or they don't get included in the final stylesheet.

```suggestion
    if (score <= bad) return 'progress-error';
```

Reply:
Done in commit 31d8c68e4c9c6781f6740aaea1236d9b4d46aac4.